cloud/edgecontroller/pkg/devicecontroller/config/kube.go

```diff
@@ -0,0 +1,81 @@
+package config
```

Comment:
Can we combine this file with https://github.com/kubeedge/kubeedge/blob/master/cloud/edgecontroller/pkg/controller/config/kube.go?

Reply:
want to keep deviceController and controller as completely seperate and independent modules, so that we can enable/disable both during comple/runtime. Thoughts ??

Reply:
Want to keep deviceController and controller completely independent of each other. So that they can enabled/disabled as per need during compile/runtime. thoughts ?

Reply:
I don't think it will confilict with your idea. A good example is kube-controller-managers.

Reply:
If we combine this file, we may need to have a common location to keep this file. I don't think having configurations for 1 module in other modules source code will be a good idea. Thougts ?

Reply:
Yes, I think we need a common place for the kube configs.